packages/wallet/backend/src/app.ts

```diff
@@ -175,6 +175,11 @@ export class App {
       isAuth,
       paymentPointerController.getById
     )
+    router.get(
+      '/payment-pointer/:url',
```

Comment:
if this is only to get external payment pointers I suggest to indicate this in endpoint name (and change to plural)

Reply:
![image](https://github.com/interledger/testnet/assets/111863110/bddfa9e6-00a6-4a3f-942e-c94012920f28)

would this be fine? or would you prefer external/payment-pointers (i don't think there will be any other external stuff though that would fall under the first / category)

Reply:
this looks fine to me